sklearn/naive_bayes.py

```diff
@@ -943,22 +943,13 @@ class BernoulliNB(BaseDiscreteNB):
     classes_ : array of shape (n_classes,)
         The classes labels.
 
-    Examples
-    --------
-    >>> import numpy as np
-    >>> rng = np.random.RandomState(1)
-    >>> X = rng.randint(5, size=(6, 100))
-    >>> Y = np.array([1, 2, 3, 4, 4, 5])
-    >>> from sklearn.naive_bayes import BernoulliNB
-    >>> clf = BernoulliNB()
-    >>> clf.fit(X, Y)
-    BernoulliNB()
-    >>> print(clf.predict(X[2:3]))
-    [3]
+    See Also
+    ----------
+    MultinomialNB: The multinomial Naive Bayes classifier is \
```

Comment:
I think the backslash at the end is not needed.

Reply:
Thanks, @rth! I tried having that bit of text on one line, but the pytest failed. Splitting into two lines made the pytest succeed, but when I submitted the PR without the backslash at the end, the lint check failed. Adding it at the recommendation of a more experienced contributor made it pass the lint test.

Is there another approach I should try instead?

Reply:
Fair enough, the docstring renders fine in any case. Thanks!